### ISSUE TITLE
Process Explorer: NVDA reads the whole table dimension every time before reading each cell value.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52562]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to Help and Click on Process Explorer

**Actual:**
NVDA reads the whole table dimension (16 rows and 4 columns) every time before reading each cell value.

**Expected:**
NVDA should read table dimension only once and should read only current position of cell while reading the cell value.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Process Explorer NVDA reads table dimensions everytime.pptx](https://github.com/Microsoft/vscode/files/2122595/A11y.VSCode.Process.Explorer.NVDA.reads.table.dimensions.everytime.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Sounds reasonable. Leaving to @RMacfarlane to decide if we should tackle this

**Comment by isidorn** [comment]
@RMacfarlane is there any chance we look into this? Not sure how to solve it though.

**Comment by RMacfarlane** [comment]
@isidorn yeah, I'll look into this. Also not sure how to solve it, but will do more investigation

**Comment by RMacfarlane** [comment]
@AccessibilityTestingTeam-TCS I haven't been able to reproduce this - how are you navigating once you have the Process Explorer open?

**Comment by isidorn** [comment]
@RMacfarlane Just a reminder regarding this. Not sure if more info is still needed?
